cluster-up/cluster/kind-k8s-sriov-1.17.0/manifests/kube-flannel.yaml

```diff
@@ -0,0 +1,531 @@
+---
```

Comment:
This file should be updated to the file you download [here](https://github.com/kubevirt/kubevirtci/pull/283/files?file-filters%5B%5D=.sh#diff-590b4724c04eebfae018481ab5d6bc9bR116).